stuff/pipelines/verification_code.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
+
+'''Create verification code to certificates.'''
+
+import logging
+import os
+import sys
+import random
+import MySQLdb
+import MySQLdb.cursors
+
+
+sys.path.insert(
+    0,
+    os.path.join(
+        os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "."))
+
+
+def generate_code(cur: MySQLdb.cursors.BaseCursor) -> str:
+    '''Generate an aleatory code'''
+    diccionary_alfabeth = {"2": 0, "3": 1, "4": 2, "5": 3,
+                           "6": 4, "7": 5, "8": 6, "9": 7,
+                           "C": 8, "F": 9, "G": 10, "H": 11,
+                           "J": 12, "M": 13, "P": 14, "Q": 15,
+                           "R": 16, "V": 17, "W": 18, "X": 19}
+    code_alfabeth = "23456789CFGHJMPQRVWX"
```

Comment:
```suggestion
    alphabet = "23456789CFGHJMPQRVWX"
```

pero igual acá, esto sería mejor moverlo como constante afuera como `_ALPHABET`.